### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bservet**
Here's the update again....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Within this tab general options are configured.
...
 * **Icon path**: In order to have Kodi display channel logos you have to copy the picons from your set-top box onto your OpenELEC machine. You then need to specify this path in this property.
-* **Update interval**: As the set-top box can also be used to modify timers, delete recordings etc. and the set-top box does not notify the Kodi installation, the addon needs to regularly check for updates (new channels, new/changed/deletes timers, deleted recordings, etc.) This property defines how often the addon checks for updates.
+* **Update interval**: As the set-top box can also be used to modify timers, delete recordings etc. and the set-top box does not notify the Kodi installation, the addon needs to regularly check for updates (new channels, new/changed/deletes timers, deleted recordings, etc.) This property defines how often the addon checks for updates. Please note that updating the recordings frequently can keep your receiver and it's harddisk from entering standby automatically.
```
Comment (phunkyfish) [inline README.md]: Extra spaces at the end of the line.

**Review by phunkyfish — CHANGES_REQUESTED** [review]

**Comment by phunkyfish** [comment]
Once this is cleaned up and approved you will also want to have a similar PR for the Leia branch.

**Comment by phunkyfish** [comment]
Can you squash this into a single commit?

Alternatively close this PR and create a new one with a single commit.

**Comment by bservet** [comment]
Ok, this commit has the spaces so'll close it as I still haven't found the squash option.

**Comment by phunkyfish** [comment]
Ok, so just create a new one for matrix with a single commit. The Leia one is OK.

**Pull request closed without merging**